doc/topology.md

```diff
@@ -114,9 +114,11 @@ are removed on [`cleanup_expelled`](/doc/scenario.md#cleanup_expelled) step.
 ## Advertise URIs changing
 
 It is possible to change the advertise URIs of the servers. However, this should be done carefully.
+You can read about manually changing the URIs here:
```

Comment:
You can read about manually changing the URIs in the [troubleshooting doc](https://tarantool.io/en/doc/latest/book/cartridge/troubleshooting/#i-want-to-run-an-instance-with-a-new-advertise-uri).